tools/replay_invocation/replay_invocation.go

```diff
@@ -181,5 +194,27 @@ func main() {
 			break
 		}
 	}
+
+	// Print invocation log chunks.
+	// TODO: replay these as synthetic progress events.
+	if *printLogs {
+		logsBlobstorePrefix := eventlog.GetEventLogPathFromInvocationIdAndAttempt(*invocationID, uint64(*attemptNumber))
+		log.Infof("Fetching log chunks from %s_*", logsBlobstorePrefix)
+		for i := uint16(0); ; i++ {
```

Comment:
Is there a reason to read and print the chunks separately? If not we can use `chunkstore.Read`

Reply:
Done. It turned out to be a little trickier than I had hoped, because I had to decide on a good buffer size for the reads.